db/migrate/20150206201847_add_site_id_to_storytime_post.rb

```diff
@@ -0,0 +1,11 @@
+class AddSiteIdToStorytimePost < ActiveRecord::Migration
+  def change
+    add_column :storytime_posts, :site_id, :integer
+
+    site = Storytime::Site.first
```

Comment:
As soon as I wrote that, I realized that Storytime::Post.all will be empty in that case as well, so feel free to ignore that previous comment.